src/harness/dtRunner.ts

```diff
@@ -0,0 +1,62 @@
+/// <reference path="harness.ts"/>
+/// <reference path="runnerbase.ts" />
+class DefinitelyTypedRunner extends RunnerBase {
+    private static readonly testDir = "../DefinitelyTyped/types/";
+
+    public workingDirectory = DefinitelyTypedRunner.testDir;
+
+    public enumerateTestFiles() {
+        return Harness.IO.getDirectories(DefinitelyTypedRunner.testDir);

```

Comment:
It might be worth throwing an error if `DefinitelyTypedRunner.testDir` is empty/does not exist (rather than the usual behavior of `getDirectories`, which I believe just produces `undefined` or an empty list). After all, without that, an empty or missing folder is a passing DT suite, which could be misleading if you run this with `DefinitelyTyped` cloned in the wrong location.

Reply:
Turns out getDirectories throws already, which I think is good enough.

Reply:
Why does a function called `enumerateTestFiles` return a list of directories?